src/test/java/org/openstreetmap/atlas/checks/validation/intersections/IntersectingBuildingsCheckTest.java

```diff
@@ -103,17 +103,16 @@ public void testSmallIntersectionBuildingsAtlasWithIncreasedIntersectionLowerLim
     }
 
     @Test
-    public void testSmallIntersectionBuildingsAtlasWithSmallOverlapLowerLimit()
```

Comment:
Why is this test case being removed?

Reply:
This test is no longer part of what the check looks for. The check now looks for containment instead of overlap in the case this test presents. `testContainsBuildingAtlas()` replaces this test.

Reply:
That is fine, but I'd suggest then cleaning up `IntersectingBuildingsTestCaseRule` by removing `smallIntersectionBuildingsAtlas` method and the corresponding variable.